tests/assets/testkey.txt

```diff
@@ -0,0 +1,5 @@
+abc def ghi jkl mno pqr stu vwx yz1@3$
```

Comment:
I've fixed the majority of the issues. There is now an issue with "GetKeys". This PR doesn't impact that. @mradcliffe - your previous PR passed in a set of Key IDs. Can you investigate?

I am still having an issue with the PHPUnit test for "GetKeysByStorageType". I'm not sure why and will need to investigate more later or have someone pick this up due to time constraints.

I'm going to merge this PR in now to allow others to make more progress